imu_x: 0.3
imu_y: 0.0
imu_z: 0.0
imu_ax: 0.0
imu_ay: 0.0
imu_az: 0.0

left_laser_x: 0.22
left_laser_y: 0.26
left_laser_z: 0.0
left_laser_ax: 0.0
left_laser_ay: -0.03
left_laser_az: 3.12

right_laser_x: 0.24
right_laser_y: -0.35
right_laser_z: 0.0
right_laser_ax: 0.0
right_laser_ay: 0.23
right_laser_az: 3.14
